JASP-Desktop/components/JASP/Controls/VariablesForm.qml

```diff
@@ -42,8 +40,8 @@ Item
 
 			property double	_lastListWidth:					0
 	
-	property int preferredHeight:	implicitHeight
-	property int preferredWidth:	implicitWidth
+	property int preferredHeight:	jaspTheme.defaultVariablesFormHeight
```

Comment:
So, what is implicitHeight and width set to now?

Shouldn't it be:
```
property int preferredHeight:	implicitHeight
property int preferredWidth:	implicitWidth
implicitHeight:	jaspTheme.defaultVariablesFormHeight
implicitWidth:	form.width
```

Reply:
A VariablesForm is normally used in a GridLayout (in the Form). In this case the preferredHeight & preferredWidth should be enough. But a VariablesForm might be used in  a sub-item that has no GridLayout. In that case, what count are the height and width properties.
I don't see why we should set the implicitWidth & implicitHeight